### PR TITLE
Make about me field Private to User

### PR DESCRIPTION
**Pull request by tharminhtet**
Make about me field in profile page editable only to the related User.

### CODE REVIEW

File: src/test/java/codeu/controller/ProfileServletTest.java [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+    Mockito.when(mockMessageStore.getMessagesForUser(fakeUserId))
+      .thenReturn(fakeMessageList);
+
+
+    profileServlet.doGet(mockRequest, mockResponse);
+
+    Mockito.verify(mockRequest).setAttribute("username", "test_user");
+    Mockito.verify(mockRequest).setAttribute("messages", fakeMessageList);
+    Mockito.verify(mockRequestDispatcher).forward(mockRequest, mockResponse);
+  }
```
Comment (sherryx8) [inline src/test/java/codeu/controller/ProfileServletTest.java:72]: You could add testing for private edit about me field functionality

File: src/main/java/codeu/controller/ProfileServlet.java [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+  {
+    String requestUrl = request.getRequestURI();
+    String userName = requestUrl.substring("/users/".length());
+    //TODO: Change to acutal about me, and handle null condition. Thar Min Htet
+    String aboutMe = "This is fake about Me.";
+
+    User user = userStore.getUser(userName);
+    UUID userId = user.getId();
+    List<Message> userMessages = messageStore.getMessagesForUser(userId);
+
```
Comment (sherryx8) [inline src/main/java/codeu/controller/ProfileServlet.java:60]: if URL of a user that does not exist is given, the page is just blank, maybe for future, we can have an error message that says the user does not exist.

**Review by sherryx8 — COMMENTED** [review]
Works correctly! I think it would be good to add testing for the private edit about me field functionality

File: src/main/java/codeu/controller/ProfileServlet.java (hunk 2 above)
Comment (tharminhtet) [reply src/main/java/codeu/controller/ProfileServlet.java:60]: Noted

File:  [hunk 3]
```diff
@@ -0,0 +1,370 @@
...
+Author: Kevin Workman <kevinaworkman@gmail.com>
+Date:   Tue Feb 6 12:52:16 2018 -0800
+
+    Update README.md
+
+[33mcommit 80fa7059e16763ba1497b327d304a42a7c47138e[m
+Author: Kevin Workman <kevinaworkman@gmail.com>
+Date:   Tue Feb 6 12:47:49 2018 -0800
+
+    Initial commit.
```
Comment (TsueiCat) [inline ]: delete this stuff!

**Review by TsueiCat — COMMENTED** [review]
nice work!

**Pull request closed without merging**